ush/global_nceppost.sh

```diff
@@ -350,13 +350,15 @@ export DD=`echo $VDATE | cut -c7-8`
 export HH=`echo $VDATE | cut -c9-10`
 
 cat > itag <<EOF
-$GFSOUT
-${MODEL_OUT_FORM}
-${GRIBVERSION}
-${YY}-${MM}-${DD}_${HH}:00:00
-GFS
-$FLXINP
-$D3DINP
+&read_itag
```

Comment:
@kayeekayee This is a legacy script. No updates are needed. Please revert the changes.